grails-app/views/order/editAdjustment.gsp

```diff
@@ -0,0 +1,103 @@
+<html>
+<head>
+	<meta http-equiv="Content-Type" content="text/html; charset=UTF-8" />
+	<meta name="layout" content="custom" />
+	<g:set var="entityName" value="${warehouse.message(code: 'default.comment.label', default: 'Comment').toLowerCase()}" />
+	<title><warehouse:message code="default.add.label" args="[entityName]" /></title>
+</head>
+
+<body>
+
+	<div class="body">
+		<g:if test="${flash.message}">
+			<div class="message">
+				${flash.message}
+			</div>
+		</g:if>
+		<g:hasErrors bean="${orderInstance}">
+			<div class="errors">
+				<g:renderErrors bean="${orderInstance}" as="list" />
+			</div>
+		</g:hasErrors>
+		<g:hasErrors bean="${orderAdjustment}">
+			<div class="errors">
+				<g:renderErrors bean="${orderAdjustment}" as="list" />
+			</div>
+		</g:hasErrors>
+
+		<div class="dialog">
+			<g:render template="summary" model="[orderInstance:orderInstance]" />
+			<div class="box">
+				<h2><warehouse:message code="default.add.label" args="[entityName]" /></h2>
+				<g:form action="saveAdjustment">
+					<g:hiddenField name="id" value="${orderAdjustment?.id}" />
+
+					<table>
+						<tbody>
+							<tr class="prop">
+								<td valign="top" class="name"><label><g:message code="order.label"/></label></td>
+								<td valign="top" class="value ${hasErrors(bean: orderAdjustment, field: 'order', 'errors')}">
+									<g:hiddenField name="order.id" value="${orderInstance?.id}" />
+									${orderInstance?.orderNumber}
+									${orderInstance?.name}
+
+								</td>
+							</tr>
+%{--							<tr class="prop">--}%
```

Comment:
Redundant code

Reply:
This is for the order item association which I will add back in soon. I just need to comment it out for the demo because it's confusing.

Reply:
ok